versions.props

```diff
@@ -12,6 +12,9 @@ org.apache.commons:commons-lang3 = 3.9
 com.fasterxml.jackson.*:* = 2.10.0
 com.github.ben-manes.caffeine:caffeine = 2.7.0
 org.apache.arrow:arrow-vector = 0.14.1
+org.apache.flink:* = 1.10.0
+com.netflix.spectator:* = 0.103.0
```

Comment:
I'm not sure that we want to use Spectator. Maybe removing metrics initially would be a good way to simplify the PR.

Reply:
Addressed with all Spectator related logic removed